### PR TITLE
feat(curriculum): add English curriculum block 8.1 sharing progress and accomplishments

### PR DESCRIPTION
**Pull request by miyaliu666**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/6610056e633ce41466b6c5bc.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+challengeType: 19
+dashedName: task-3
+---
+
+<!--
+AUDIO REFERENCE:
+Sarah: Sure. This week, I worked on the user interface design. I designed a new layout for the app, and I created some mockups to show how it will look.
+-->
+
+# --description–
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/6610056e633ce41466b6c5bc.md:13]:
```suggestion
# --description--
```



This got malformed somehow

**Review by moT01 — COMMENTED** [review]
A couple inconsistencies in the titles

**Review by moT01 — APPROVED** [review]
I committed my suggestions and a fix to one of the answers.

I didn't go over this with a fine-toothed comb, but I looked over it pretty good. Mainly, I looked at the files - as far as I could tell, the backticks were in all the right places, the questions and fill in the blanks all had the correct answers, and I didn't see any grammar issues.

I would be fine getting this in. We still have to add the videos where we will have another chance to make improvements or catch issues. LGTM @miyaliu666 🎉

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/66125530a1a9e92e0e08d594.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+`_. Learning more about our customers is essential. I _ your _. Can you tell me about your _ goals?`
+
+## --blanks--
+
+`Excellent`
+
+### --feedback--
+
+It is used by Maria to positively acknowledge the information shared.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/66125530a1a9e92e0e08d594.md:37]:
```suggestion
It is used by Maria to positively acknowledge the information shared. This word is capitalized.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/661256fe823f142fb9858beb.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Brian: Of course. By the end of the quarter, I plan to have the documentation I’m working on fully updated, and I intend to improve our customer support process based on the training I received.
+-->
+
+# --description--
+
+Practice time!
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/661256fe823f142fb9858beb.md:15]:
```suggestion
Listen to the audio and answer the question.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/661269c4ccdd4132db7517b8.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Brian: Thanks, Maria. I appreciate your guidance.
+-->
+
+# --description--
+
+Practice time!
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/661269c4ccdd4132db7517b8.md:15]:
```suggestion
Listen to the audio and answer the question.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/66126a21821998335a86a34b.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Entire dialogue
+-->
+
+# --description--
+
+This task involves summarizing a professional performance review conversation. It's important to capture the essence of such discussions, which often include updates on recent work, feedback, goal setting, and offers of support.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/66126a21821998335a86a34b.md:15]:
```suggestion
Listen to the dialogue and fill in the blanks with the correct words.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/661271c2b16aff3870604148.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Brian: Hey, Sarah! It's been a while since we hung out like this.
+-->
+
+# --description--
+
+This task assesses your understanding of the time frame mentioned by Brian in his conversation with Sarah.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/661271c2b16aff3870604148.md:15]:
```suggestion
When people say `it's been a while`, they mean a long time has passed between today and the last time something happened. 
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/6612757ffdc16b3a22b08427.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Brian: That sounds awesome! I tested the app for bugs. We found a few, but the good news is we fixed them all. It seems pretty solid now.
+-->
+
+# --description--
+
+Practice time!
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/6612757ffdc16b3a22b08427.md:15]:
```suggestion
Listen to the audio and complete the sentence.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/6612762a058bb43b960e91ca.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Brian: That sounds awesome! I tested the app for bugs. We found a few, but the good news is we fixed them all. It seems pretty solid now.
+-->
+
+# --description--
+
+Practice time!
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/6612762a058bb43b960e91ca.md:15]:
```suggestion
Listen to the audio and answer the question.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/66127755a52efa3c9a73065b.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Sarah: Nice work! I also attended a UX design workshop last month. I learned some great techniques for user research. I've applied them to our project, and it helped improve the user experience.
+-->
+
+# --description--
+
+Practice time.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/66127755a52efa3c9a73065b.md:15]:
```suggestion
Listen to the audio and complete the sentence.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/661278160653ee3d9040ed68.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Sarah: Nice work! I also attended a UX design workshop last month. I learned some great techniques for user research. I've applied them to our project, and it helped improve the user experience.
+-->
+
+# --description--
+
+Practice time!
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/661278160653ee3d9040ed68.md:15]:
```suggestion
Listen to the audio and answer the question.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/6612797faf03663ef83f4459.md [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+
+`_ to go! It looks like _ making great progress.`
+
+## --blanks--
+
+`Way`
+
+### --feedback--
+
+It is used here as part of an expression of congratulations or approval for good work or progress.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/6612797faf03663ef83f4459.md:31]:
```suggestion
It is used here as part of an expression of congratulations or approval for good work or progress. This word is capitalized.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/661279c8d3bf0f3f6f23f21f.md [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Entire dialogue
+-->
+
+# --description--
+
+This task involves summarizing a professional conversation between Brian and Sarah, focusing on their recent activities and achievements.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/661279c8d3bf0f3f6f23f21f.md:15]:
```suggestion
Listen to the dialogue and answer the question.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/661279c8d3bf0f3f6f23f21f.md [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+
+# --question--
+
+## --text--
+
+What is the summary of Brian and Sarah's conversation?
+
+## --answers--
+
+Brian and Sarah discussed their recent work, including app testing, UX design workshop attendance, and client meetings, all leading to positive progress on their project.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-progress-and-accomplishments/661279c8d3bf0f3f6f23f21f.md:25]:
```suggestion
They discussed their work, a UX design workshop, and client meetings.
```

**Review by nieldakarla — CHANGES_REQUESTED** [review]
Excellent job, @miyaliu666 ! 


I've added a couple changes, specially replacing "practice time". I'll request changes but once these are implemented I'll approve the PR.

**Review by nieldakarla — APPROVED** [review]

**Pull request merged**